vnext/ReactWindowsCore/Modules/UIManagerModule.cpp

```diff
@@ -26,7 +26,9 @@ UIManager::UIManager(std::vector<std::unique_ptr<IViewManager>> &&viewManagers,
 
 UIManager::~UIManager() {
   m_nodeRegistry.removeAllRootViews([this](int64_t rootViewTag) { removeRootView(rootViewTag); });
-
+  for (auto it = m_nodeRegistry.getAllNodes().begin(); it != m_nodeRegistry.getAllNodes().end(); it++) {
```

Comment:
I'd rather have us give the m_nodeRegistry a lambda to execute over its elements than to tightly couple it to the type of a private member, so maybe something like:
```cpp
m_nodeRegistry.ForAllNodes([] (int tag, shadow_ptr const&) {  DropView(tag, false, true); });
```